### PR TITLE
Release 2021.09.0

### PR DESCRIPTION
**Pull request by christinechoy-okta**
## Description:
- **What's changed?** Adding Release Notes for 2021.09.0
- **Is this PR related to a Monolith release?** Yes, Release Notes 2021.09.0

### Resolves:

* [OKTA-423533](https://oktainc.atlassian.net/browse/OKTA-423533)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/release-notes/2021/index.md [hunk 1]
```diff
@@ -1,6 +1,45 @@
...
+
+#### Dynamic Issuer Mode is EA in Preview
+
+An authorization server's issuer URL can be used to validate whether tokens are issued by the correct authorization server. You can configure the issuer URL to be either the Okta subdomain (such as company.okta.com) or a custom domain (such as sso.company.com). See [Property details](/docs/reference/api/authorization-servers/#authorization-server-properties).
+
+When there are applications that use Okta's subdomain and other applications that use the custom domain, the issuer validation breaks because the value is hard-coded to one domain or the other.
+
+With Dynamic Issuer Mode, the issuer value in minted tokens is dynamically updated based on the URL that is used to initiate the original authorize request. For example, if the authorize request is https://sso.company.com/api/v1/authorize, the issuer value is https://sso.company.com. See [Client application settings](/docs/reference/api/apps/#settings-10).
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2021/index.md:25]:
```suggestion
With Dynamic Issuer Mode, the issuer value in minted tokens is dynamically updated based on the URL that is used to initiate the original authorize request. For example, if the authorize request is `https://sso.company.com/api/v1/authorize`, the issuer value is `https://sso.company.com`. See [Client application settings](/docs/reference/api/apps/#settings-10).
```

File: packages/@okta/vuepress-site/docs/release-notes/2021/index.md [hunk 2]
```diff
@@ -1,6 +1,45 @@
...
+
+Dynamic Issuer Mode helps with:
+* Split deployment use cases
+* Migration use cases when customers migrate from the Okta domain to a custom domain
+* Support with multiple custom domains <!--OKTA-411419-->
+
+#### The Subscriptions API is GA in Production
+
+The [Subscriptions API](/docs/reference/api/admin-notifications/) is now Generally Available in Production. The Subscriptions API provides operations to manage email subscription settings for Okta administrator notifications.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2021/index.md:34]: Don't repeat the title in the content

```suggestion
The [Subscriptions API](/docs/reference/api/admin-notifications/) provides operations to manage email subscription settings for Okta administrator notifications.
```

**Review by susanharper-okta — COMMENTED** [review]
couple things.

**Review by brianduffield-okta — APPROVED** [review]
LGTM - Just the comment in Slack.

**Pull request merged**